src/server/service/kv.rs

```diff
@@ -480,9 +485,14 @@ impl<E: Engine, L: LockManager, F: KvFormat> Tikv for Service<E, L, F> {
         forward_unary!(self.proxy, coprocessor, ctx, req, sink);
         let source = req.get_context().get_request_source().to_owned();
         let resource_control_ctx = req.get_context().get_resource_control_context();
+        let mut resource_group_priority = ResourcePriority::unknown;
         if let Some(resource_manager) = &self.resource_manager {
             resource_manager.consume_penalty(resource_control_ctx);
+            let priority = resource_manager
```

Comment:
after https://github.com/tikv/tikv/pull/15929. I think we can directly use the `override_priority` from context.

Reply:
fixed